web/themes/contrib/civictheme/civictheme_library/components/03-organisms/mobile-navigation/mobile-navigation.scss

```diff
@@ -69,6 +69,7 @@ $ct-mobile-navigation-spacing: ct-spacing(2);
       border-left-style: solid;
       padding: $ct-mobile-navigation-spacing;
       border-radius: 0;
+      background-color: transparent;
```

Comment:
please move colour override to the `@include ct-component-theme` section below. We manage all theme-related settings in one section.

Reply:
Thanks. Moved to component theme section.